vortex-layout/src/layouts/chunked/reader.rs

```diff
@@ -91,11 +88,11 @@ impl ChunkedReader {
                             )
                             .await?;
 
-                        Some(StatsTable::try_new(
+                        Some(StatsTable::try_new_unchecked(
```

Comment:
no need to check the dtype because we use the `DType` to read the stats layout.

Reply:
Why no need to check? We cannot trust the contents of a file